Gems/ScriptCanvas/Code/Editor/View/Windows/Tools/UpgradeTool/Scanner.cpp

```diff
@@ -17,6 +17,8 @@
 
 namespace ScannerCpp
 {
+    using namespace ScriptCanvas;
+
```

Comment:
Repeat: the `using namespace ScriptCanvas;` call needs to be moved to function scope or removed altogether